janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server.yaml

```diff
@@ -2,8 +2,11 @@ host: 0.0.0.0
 port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+graphManager: org.janusgraph.graphdb.management.JanusGraphManager
```

Comment:
I don't think users would do a mix of `JanusGraphManager` and individually configured graphs, even though it is possible to do within one YAML. Mixing the 2 would probably be confusing for existing users. I think the `graphManager` should be a separate YAML file example.

Reply:
In my mind this should be the default yaml we want to point users towards, i.e. server configured using the JGM, the CMG, and leave the default configured graph `graph`

Reply:
It very well could become the new default, but I'd think that would be a JG 1.0 item -- breaking change, this is the main way of doing it going forward.

Why leave the default `graph`? Now you're creating 2 graphs by default when you start the pre-packaged distribution, and it muddies the new user experience.

Another thought, how about something like this instead to better align the config graph with the graph manager:
```
graphManager: {
    class: org.janusgraph.graphdb.management.JanusGraphManager,
    config: conf/janusgraph-configurationmanager.properties
}
```

Reply:
This is a good idea, but the benefits of leaving the configuration graph as a `graph` are that you can act on it directly as a globally bound graph and I am pretty sure that this would require a change to the way TinkerPop Settings class digests a `graphManager` option

Reply:
> Why leave the default graph? Now you're creating 2 graphs by default when you start the pre-packaged distribution, and it muddies the new user experience.

The first graph is the continued graph the user used to and can continue using as their database, and the new graph _is_ the configuration management graph, to be consumed specifically for its own set of purposes.


Do you still want me to create a new yaml? I am OK with that, but I think that leaving the default `graph` still is a better UX.

Reply:
Good point, if the `graphManager` config block requires changes in TinkerPop, don't worry about that for now.

I'd rather see a separate yaml for this configuration. I don't forsee developers mixing the 2 approaches, and I think it would be confusing for new users.